packages/webapp/src/containers/Finances/EditSale/index.jsx

```diff
@@ -136,63 +66,30 @@ class EditSale extends Component {
     return cropVarietyOptions;
   };
 
-  render() {
-    let managementPlans = this.props.managementPlans || [];
-    const cropVarietyOptions = this.getCropVarietyOptions(managementPlans);
-    return (
-      <div className={defaultStyles.financesContainer}>
-        <PageTitle
-          title={this.props.t('SALE.EDIT_SALE.TITLE')}
-          style={{ marginBottom: '24px' }}
-          onGoBack={() => history.back()}
-        />
-        <span className={defaultStyles.dateContainer}>
-          <label>{this.props.t('SALE.EDIT_SALE.DATE')}</label>
-          <DateContainer
-            style={defaultStyles.date}
-            custom={true}
-            date={this.state.date}
-            onDateChange={(date) => this.setState({ date })}
-          />
-        </span>
-        <SaleForm
-          model="financeReducer.forms.editSale"
-          cropVarietyOptions={cropVarietyOptions}
-          chosenOptions={this.state.chosenOptions}
-          handleChooseCrop={this.handleChooseCrop}
-          onSubmit={this.handleSubmit}
-          quantityUnit={this.state.quantity_unit}
-          footerOnClick={() => this.setState({ showModal: true })}
-          footerText={this.props.t('common:DELETE')}
-          currencySymbol={this.state.currencySymbol}
-        />
-        <ConfirmModal
-          open={this.state.showModal}
-          onClose={() => this.setState({ showModal: false })}
-          onConfirm={() => {
-            this.props.dispatch(deleteSale(this.props.sale));
-          }}
-          message={this.props.t('SALE.EDIT_SALE.DELETE_CONFIRMATION')}
-        />
-      </div>
-    );
-  }
+  const cropVarietyOptions = getCropVarietyOptions();
```

Comment:
I JUST remembered that delete functionality is about to be added! So I think at some point you might be able to get to EditSale.jsx after deleting all your management plans. So best to include the same check here as well (see [comment above](https://github.com/LiteFarmOrg/LiteFarm/pull/2554/files#r1161350919))

Reply:
done